lib/rubygems/package.rb

```diff
@@ -385,7 +385,13 @@ def extract_tar_gz io, destination_dir, pattern = "*" # :nodoc:
           FileUtils.chmod entry.header.mode, destination
         end if entry.file?
 
-        File.symlink(entry.header.linkname, destination) if entry.symlink?
+        if entry.symlink?
+          begin 
+            File.symlink(entry.header.linkname, destination)
+          rescue NotImplementedError
+            # Not all Rubies support symlinks
```

Comment:
@segiddins I should mention I did think about notion that a symlink could lead to a copy too but symlinks may or may not refer to regular files and may or may not link to something within the gem (although that one is a bit weird).  So I think the avenue will lead to strange bug reports down the line.